.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Description
+[Tell us about your new feature, improvement, or fix! If relevant, please supplement the PR with images.]
+
+
+## Checklist
+[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
+- [ ] [PEP8 compliance](https://www.python.org/dev/peps/pep-0008/)
+- [ ] Docstrings for all functions
```

Comment:
I view this more as a getting-started pointer, rather than sending people out with homework ;)